### ISSUE TITLE
syntax highlight UDO names

### ISSUE THREAD

**Issue by tgrey1** [rorywalsh/csoundSublime#1]
It would be useful if instances of a UDO's name were highlighted a custom color.  I imagine this is possible somehow, probably similar to how method/function name highlighting would work in most other languages.

**Comment by rorywalsh** [comment]
I don't think the syntax highlighting is dynamic, therefore this isn't possible within the current framework. But I guess other plugins might do this using a different mechanism. Could be worth taking a look.

**Comment by tgrey1** [comment]
You're right, I saw the function declaration highlighted, and hastily assumed that any instances calling the method would be highlighted too.  It seems like sublime _should_ be able to do this, one way or another... multi-selection seems to show that it can be contextually aware of it's content.

I found this with some quick searching:
http://www.sublimetext.com/forum/viewtopic.php?f=3&t=7932

I don't know the first thing about sublime, I'm a brand new user... and the only reason I have to learn it would be this, but perhaps if I get a chance to learn a bit I can figure out if that link is actually helpful here or not ;)  Also, was the highlighting something Takahiko originally worked on?  If so, maybe asking him for help or suggestions might be a good idea too.

I think this would be a very useful addition, and even if there's no immediate plans for it, I'd leave the suggestion open for future tracking, but that's obviously up to you.
